.github/workflows/main.yml

```diff
@@ -11,6 +11,22 @@ on:
   pull_request:
 
 jobs:
+  close-issues:
```

Comment:
I would define this is a separate workflow and use a schedule cron trigger like in the github example.

Reply:
Agree, updated in the new commit.